### PR TITLE
[App][CI] Delete apps older than 1 hours CI account

### PR DESCRIPTION
**Pull request by manskx**
## What does this PR do?

This PR deletes apps in the CI account that are older than 1 hour. This account is used for CI tests only so no test should run for more than one hour.

Fixes #\<issue_number>

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @carmocca @akihironitta @borda

### CODE REVIEW

File: src/lightning_app/testing/testing.py [hunk 1]
```diff
@@ -458,11 +458,24 @@ def wait_for(page, callback: Callable, *args, **kwargs) -> Any:
...
+    except ApiException as e:
+        print(f"Failed to delete {app_name}. Exception {e}")
+
+
 def delete_cloud_lightning_apps():
     """Cleanup cloud apps that start with the name test-{PR_NUMBER}-{TEST_APP_NAME}.
 
     PR_NUMBER and TEST_APP_NAME are environment variables.
     """
+    import datetime
```
Comment (awaelchli) [inline src/lightning_app/testing/testing.py:478]: I suggest to move the import to the top of the file

**Review by awaelchli — APPROVED** [review]

File: src/lightning_app/testing/testing.py [hunk 2]
```diff
@@ -458,11 +458,24 @@ def wait_for(page, callback: Callable, *args, **kwargs) -> Any:
...
+def _delete_lightning_app(client, project_id, app_id, app_name):
+    print(f"Deleting {app_name} id: {app_id}")
+    try:
+        res = client.lightningapp_instance_service_delete_lightningapp_instance(
+            project_id=project_id,
+            id=app_id,
+        )
+        assert res == {}
+    except ApiException as e:
+        print(f"Failed to delete {app_name}. Exception {e}")
```
Comment (Borda) [inline src/lightning_app/testing/testing.py:470]:
```suggestion
    except ApiException as ex:
        print(f"Failed to delete {app_name}. Exception {ex}")
```

**Review by Borda — APPROVED** [review]

**Comment by Borda** [comment]
@tchaton, mind checking this one? :otter:

**Review by tchaton — APPROVED** [review]
LGTM !

**Comment by manskx** [comment]
@lantiga I need your review to merge this

**Pull request merged**